src/PatientChecking/PatientChecking/Views/Appointment/Index.cshtml

```diff
@@ -152,4 +167,8 @@
         url = url.replace("optionSort", selectedSortValue).replace("pagingOption", selectedPagingValue).replace("currentPage", 1);
         window.location.href = url;
     });
+    $("#alert-message").fadeTo(3000, 500).slideUp(500, function () {
+        $("#alert-message").slideUp(1000);
+        $("#alert-message").attr('class', '');
+    });
```

Comment:
The jquery selector need to be declared at top of the module. anyway, please skip, I will let another people fix it.